modules/ecs6-class/point.js

```diff
@@ -1,14 +1,25 @@
 class Point {
     constructor({x=0, y=0}={}) {
+     if (!(typeof (x) === "number") || !(typeof (y) === "number"))
+          throw new Error('x and y must be of the number type')
```

Comment:
what happens if one is not a number? 
I expect to get => if x is not a number :  new Error(the parameter x must be a number) and if y is not a number: new Error(parameter y is not a number) if both are not numbers: new Error(both parameter must be type of number)